go/consensus/genesis/genesis.go

```diff
@@ -41,5 +43,14 @@ func (g *Genesis) SanityCheck() error {
 		return fmt.Errorf("consensus: sanity check failed: timeout commit must be >= 1ms")
 	}
 
+	// Check for duplicate entries in the pk blacklist.
+	m := make(map[signature.PublicKey]bool)
+	for _, v := range g.PublicKeyBlacklist {
+		if m[v] {
```

Comment:
Maybe also check if the key is already statically blacklisted and fail the sanity check? As otherwise `Blacklist()` will fail later.